LICENSE.md

```diff
@@ -1,8 +1,7 @@
-Copyright (c) 2013-2019 Anthony Eden
+# Copyright (c) 2012-2025 DNSimple Corporation
```

Comment:
How did 2013 became 2012? 😉 

To be fair, we could/should copy two lines:

Copyright (c) 2013-2019 Anthony Eden
Copyright (c) 2020-2025 DNSimple Corporation

It doesn't play very well with MD formatting and rule. We could keep it as LICENSE.txt or simply LICENSE as well.

(I know others do it https://github.com/stripe/stripe-ruby/blob/master/LICENSE)

Reply:
https://github.com/dnsimple/bugsnag-erlang/pull/19/commits/ed4a323a0d22e08b93351c53584b03f758d5452c